Tests/fixtures/tablet-3.yml

```diff
@@ -9533,165 +9533,120 @@
 -
   user_agent: Mozilla/5.0 (Linux like Android; en_US) AppleWebkit/538.1 PocketBook/641 (screen 758x1024; FW U641.5.18.72) Mobile
   os:
-    name: Android
+    name: GNU/Linux
     version: ""
     platform: ""
-  client:
-    type: browser
-    name: Android Browser
-    version: ""
-    engine: WebKit
-    engine_version: "538.1"
+  client: null
```

Comment:
there's a browser there, and it's called Browser

name `PocketBook Browser`
engine WebKit
engine_version 538.1